frontend/src/app/events/pages/event-create/event-create.page.ts

```diff
@@ -13,44 +13,112 @@ import {HttpErrorResponse} from "@angular/common/http";
   styleUrls: ['./event-create.page.scss'],
 })
 export class EventCreatePage implements OnInit {
+  @ViewChild('eventImage') eventImage: ElementRef;
+
   form: FormGroup;
+  isFree = true;
+  imageError = false;
   constructor(private formBuilder: FormBuilder, private eventService: EventService, private router: Router,
               private loadingController :LoadingController, private alertController: AlertController) {
     this.form = this.formBuilder.group({
-      name: ['', [Validators.required,Validators.pattern('[a-zA-Z ]*')]],
-      price: ['', [Validators.required,Validators.pattern('[0-9₺$€]*')]],
-      owner: ['', [Validators.required,Validators.pattern('[a-zA-Z ]*')]],
+      medias: this.formBuilder.array([
+        this.formBuilder.control('', Validators.required)
+      ]),
+      name: ['', [Validators.required]],
+      date: ['', Validators.required],
+      duration: this.formBuilder.group({
+        length: ['', Validators.required],
+        unit: ['', Validators.required]
+      }),
+      locationConstruct: this.formBuilder.group({
+        locationName: ['', Validators.required]
+      }),
+      isFree: [true, Validators.required],
+      price: this.formBuilder.group({
+        amount: [0],
+        currency: ['X']
+      }),
       description: ['', [Validators.required,Validators.minLength(20)]],
-      date: ['', Validators.required]
+      artists: this.formBuilder.array([
+        this.formBuilder.control('')
+      ])
     });
   }
 
   ngOnInit() {
   }
 
   createEvent() {
+
+    console.log(this.form.value);
     this.presentLoading();
 
+
     this.eventService
       .post(this.form.value)
       .subscribe(
         message => {
           this.router.navigate(['/feed']);
-          this.loadingController.dismiss();
         },
         error => {
-          console.log(error);
+          this.handleError(error)
+        },() => {
           this.loadingController.dismiss();
         }
       );
+
   }
 
   async presentLoading(){
```

Comment:
should remove this actually => any loading controller should not exist in our app. I decided to remove it because it does not disappear in a stable manner. Instead please use *ngIf and ion-spinner, and a variable something like dataLoaded : boolean to control loading